pkgs/by-name/sw/sway-unwrapped/package.nix

```
@@ -31,6 +31,18 @@ stdenv.mkDerivation (finalAttrs: {
       inherit swaybg;
     })
 
+    # libinput-1.27 support:
+    #   https://github.com/swaywm/sway/pull/8470
+    (fetchpatch {
+      name = "libinput-1.27-p1.patch";
+      url = "https://github.com/swaywm/sway/commit/c8ea30122a0346ebf9959db194a78c8f677b2e0d.patch";
```

```suggestion
      url = "https://github.com/swaywm/sway/commit/bbadf9b8b10d171a6d5196da7716ea50ee7a6062.patch";
```
This have been merged and have in-tree commits now. We should probably use those instead